plutus-ir/test/serialization/serializeTestTerm.plc

```diff
@@ -0,0 +1,93 @@
+(let
+  (nonrec)
+  (termbind
+    (vardecl x [b a])
+    [
+      (lam
+        n
+        (fix nat (all r (type) (fun r (fun (fun nat r) r))))
+        (wrap
```

Comment:
I'm perhaps missing something, but aren't `fix` and `wrap` actually called `ifix` and `iwrap`?